bootstrap_config.py

```diff
@@ -0,0 +1,145 @@
+import os
+from pathlib import Path
+
+import requests
+from dotenv import load_dotenv
+
+from bot.constants import Webhooks, _Categories, _Channels, _Roles
+from bot.log import get_logger
+
+load_dotenv()
+log = get_logger("Config Bootstrapper")
+
+env_file_path = Path(".env.server")
+BOT_TOKEN = os.getenv("BOT_TOKEN", None)
+GUILD_ID = os.getenv("GUILD_ID", None)
+
+
+if not BOT_TOKEN:
+    message = (
+        "Couldn't find BOT_TOKEN in the environment variables."
+        "Make sure to add it to the `.env` file likewise: `BOT_TOKEN=value_of_your_bot_token`"
+    )
+    log.warning(message)
+    raise ValueError(message)
+
+if not GUILD_ID:
+    message = (
+        "Couldn't find GUILD_ID in the environment variables."
+        "Make sure to add it to the `.env` file likewise: `GUILD_ID=value_of_your_discord_server_id`"
+    )
+    log.warning(message)
+    raise ValueError(message)
+
+
+BASE_URL = "https://discord.com/api/v10"
+
+HEADERS = {"Authorization": f"Bot {BOT_TOKEN}"}
+
+
+def get_all_roles(guild_id: int | str) -> dict:
+    """Fetches all the roles in a guild."""
+    result = {}
+
+    roles_url = f"{BASE_URL}/guilds/{guild_id}/roles"
+    response = requests.get(url=roles_url, headers=HEADERS)
```

Comment:
Will do, however we wouldn't want to do that for `get_webhook` since we need to return the empty object for us to know that it doesn't exist. Or just catch it there

Reply:
That sounds fine (side note: should we return a more-explicit null?)

Reply:
I actually was planning on making it a predicate, so basically a yes or no question to whether the webhook is there or not

Reply:
Done here 0c9cd82016ee9b67f02d12f8874cfd33fdc74570